src/main/java/com/microsoft/azure/vmagent/AzureVMManagementServiceDelegate.java

```diff
@@ -214,8 +215,8 @@ public AzureVMDeploymentInfo createDeployment(
             }
             LOGGER.log(Level.INFO,
                     "AzureVMManagementServiceDelegate: createDeployment:"
-                            + " Creating a new deployment {0} with VM base name {1}",
-                    new Object[]{deploymentName, vmBaseName});
+                            + " Creating a new deployment {0} with VM base name {1}, ephemeralOSDisk={2}",
```

Comment:
I think this is not a good place to log ephemeral OS disk information.

Reply:
removed ephemeral disk OS logging here